File: yt/yt/flow/examples/java/static_table_join/static_table_join/src/main/java/tech/ytsaurus/flow/examples/statictablejoin/RunnerMain.java
package tech.ytsaurus.flow.examples.statictablejoin;

import tech.ytsaurus.flow.pipeline.SimpleRunnerProgram;

public class RunnerMain {

    private RunnerMain() {
    }

    // [BEGIN main]
    public static void main(String[] args) throws Exception {
        SimpleRunnerProgram.runPipeline(args);
    }
    // [END main]
}

File: yt/yt/flow/examples/kotlin/static_table_join/static_table_join/src/main/kotlin/tech/ytsaurus/flow/examples/statictablejoin/RunnerMain.kt
package tech.ytsaurus.flow.examples.statictablejoin

import tech.ytsaurus.flow.pipeline.SimpleRunnerProgram

object RunnerMain {

    // [BEGIN main]
    @JvmStatic
    fun main(args: Array<String>) {
        SimpleRunnerProgram.runPipeline(args)
    }
    // [END main]
}
